src/dso_api/dynamic_api/serializers.py

```diff
@@ -48,7 +48,7 @@ class DynamicSerializer(DSOSerializer):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        request = self.context["request"]
+        request = self.context.get("request")
```

Comment:
of request meegeven in alle tests, want nu laten we security compleet varen als de request vergeten wordt.